src/services/iasql.ts

```diff
@@ -706,9 +706,12 @@ export async function modules(all: boolean, installed: boolean, dbAlias: string,
   }
 }
 
-export async function install(moduleList: string[], dbAlias: string, user: any) {
+export async function install(moduleList: string[], dbAlias: string, user: any, allModules = false) {
```

Comment:
this could be an optional flag in the cli too `iasql install --all`

Reply:
Would very quickly become a bad idea as the number of modules balloons, though.